tests/integration/test_slurm.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+from jobflow_remote.testing import create_detour
```

Comment:
This import is what is causing the issue further up inside `conftest`. For now all `JFR` imports have to be inside the jobs themselves, otherwise the settings will be mangled (as you found). I'm not sure there is a perfect solution for this, beyond adding better functionality for programmatically resetting the settings (i.e., these tests could still break unit tests elsewhere in a test run)

Reply:
Thanks for pointing this out. I fixed this and also removed my "solution" in the `conftest`.